db/migrate/20170425154145_add_sti_to_middlewares.rb

```diff
@@ -0,0 +1,89 @@
+class AddStiToMiddlewares < ActiveRecord::Migration[5.0]
+  class MiddlewareDatasource < ApplicationRecord
+    self.inheritance_column = :_type_disabled
```

Comment:
Yeah, it is adding STI, but the migration uses a stripped-down version of the class (to bypass "triggers", validations, virtual attributes and so on), so it needs to be as minimal as possible, and every time the table has a "type" column it assumes STI is being used, but on this (and the spec) case, we don't want that to happen.

This should explain what I'm doing: http://blog.testdouble.com/posts/2014-11-04-healthy-migration-habits.html